### PR TITLE
cosmetic list pied piper

### PR DESCRIPTION
**Pull request by Cam1no**
I made endpoint, but I dont want to use that endpoint why we use GraphQL .

pied piper

### CODE REVIEW

File: app/views/react/root.html.slim [hunk 1]
```diff
@@ -1 +1,2 @@
+= stylesheet_link_tag href='https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css'
```
Comment (Cam1no) [inline app/views/react/root.html.slim]: I dont know how to install bluma.

I tried `yarn add bluma ` but, that not good coz after install that, webpakcer failure......................

File: app/controllers/cosmetics_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class CosmeticsController < ApplicationController
```
Comment (Cam1no) [inline app/controllers/cosmetics_controller.rb]: I think we dont need controller, coz we use GraphQL, but first, I will  change GraphQL endpoints next

File: app/controllers/cosmetics_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  def index
+    cosmetics = Cosmetic.limit(20)
+    render json: cosmetics.to_json
```
Comment (Cam1no) [inline app/controllers/cosmetics_controller.rb]: I dont know why I can write English without dictionally . 
hahaha you know Im probably drunk .see you later bye

File: app/javascript/src/app/CosmeticLists/index.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                    <tr key={`cosmetic-${i}`}>
+                      <th>{`${cosmetic.name}`}</th>
+                    </tr>
+                  )
+                })
+              }
+            </tfoot>
+          </table>
+        </div>
+      ) : <div>HelloWorld</div>
```
Comment (Cam1no) [inline app/javascript/src/app/CosmeticLists/index.jsx]: hey bro , I dont use this components, do you know where is free template components withount css , I need loading components make in only styled-components

File: app/views/react/root.html.slim (hunk 1 above)
Comment (javenkn) [reply app/views/react/root.html.slim]: おけ！調べます！

File: app/javascript/src/app/CosmeticLists/index.jsx (hunk 4 above)
Comment (javenkn) [reply app/javascript/src/app/CosmeticLists/index.jsx]: 混乱している！笑笑

File: app/controllers/cosmetics_controller.rb (hunk 3 above)
Comment (javenkn) [reply app/controllers/cosmetics_controller.rb]: 面白い！！！！！笑笑

File: app/controllers/cosmetics_controller.rb (hunk 2 above)
Comment (javenkn) [reply app/controllers/cosmetics_controller.rb]: ありがとうございます！

**Pull request merged**